packages/commonwealth/client/scripts/state/api/threads/deleteThreadLinks.ts

```diff
@@ -0,0 +1,44 @@
+import { useMutation } from '@tanstack/react-query';
+import axios from 'axios';
+import Thread, { Link } from 'models/Thread';
+import app from 'state';
+import { updateThreadInAllCaches } from './helpers/cache';
+
+interface deleteThreadLinksProps {
+  chainId: string;
+  threadId: number;
+  links: Link[];
+}
+
+const deleteThreadLinks = async ({
+  chainId,
+  threadId,
+  links
+}: deleteThreadLinksProps) => {
+  const response = await axios.delete(`${app.serverUrl()}/linking/deleteLinks`, {
+    data: {
+      thread_id: threadId,
+      links,
+      jwt: app.user.jwt,
+    },
+  })
+
+  return new Thread(response.data.result)
```

Comment:
sooo, sometimes we use `new Thread(response...)` and sometimes we return just a raw response. Also somewhere else in the code we use `new` and sometimes not. What is the logic behind it? It would be nice to have some standardised way, if not in the all project, at least here in the threads area. Can you share your thoughts here and it would be ideal if everybody knows the reasoning how we should handle it in the future 🙏